test/test_dagnav.py

```diff
@@ -157,6 +151,99 @@ def test_group_create(request):
     assert model.groups_by_name['mat:plastic'] == new_group2
     assert len(model.groups) == orig_num_groups + 2
 
+def test_group_name_runtime_error():
```

Comment:
If this tag is missing, it is created by the `Model` class so we should never be in this state.

The `Group.name` property should only return `None` if the name tag isn't set.